src/Symfony/Component/Runtime/autoload.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Symfony\Component\Runtime\SymfonyRuntime;
+
+if (true === require_once dirname(__DIR__, 2).'/autoload.php') {
+    return;
+}
+
+if (!isset($_SERVER['APP_RUNTIME'])) {
+    $runtime = SymfonyRuntime::boot($_SERVER['APP_RUNTIME_OPTIONS'] ?? []);
+} elseif (is_string($runtime = $_SERVER['APP_RUNTIME'])) {
+    $runtime = $runtime::boot($_SERVER['APP_RUNTIME_OPTIONS'] ?? []);
+}
+
+if (1 === $app = require $_SERVER['SCRIPT_FILENAME']) {
+    exit(0);
+}
+
+$app = $runtime->resolve($app)();
```

Comment:
It's possible that we reach this place without `$runtime` being set. Can we throw a meaningful exception in this case?

Reply:
nope, that's not possible, see inside the "elseif"

Reply:
Yea, but I mean at this moment you are trying to call a method on maybe a number or something like this. It's not guaranteed to be an instance of `RuntimeInterface` here